_Week9/Group4/estimation.do

```diff
@@ -0,0 +1,99 @@
+********************************************************************************
+* ECON 491
+* Week 9 - Estimation
+* Group 4
+* November 14, 2022
+********************************************************************************
+
+cap prog drop projectdata_sim
+
+prog define projectdata_sim, rclass
+
+clear
+
+********************************************************************************
+* setting up inputs
+********************************************************************************
+
+{
+	
+syntax anything
+local bias: word 1 of `anything' //a dummy for whether the results are biased or not
+local cluster_size: word 2 of `anything' //inputting cluster size
+local number_of_schools: word 3 of `anything' //inputting number of schools
+local te: word 4 of `anything' //inputting treatment effect
+local takeup_rate: word 5 of `anything' //inputting takeup rate
+
+}
+
+********************************************************************************
+* data generating process
+********************************************************************************
+
+{
+
+clear
```

Comment:
Is it necessary to `clear` twice (once here and once at the top of the program)?